captainhook.json

```diff
@@ -21,15 +21,6 @@
             {
                 "action": "composer test",
                 "options": []
-            },
```

Comment:
These need putting back in, along with the scripts and dependencies noted below. They run linting and static analysis on commit, to reduce the feedback loop.

Reply:
We've reverted these because we aren't running linting for this merge

Reply:
Ok, is there a plan to put them back in straight afterwards?

Reply:
yes, we will probably raise tickets to fix linting and add strict types and add to the backlog (Aaron depending for priority)